cg/server/admin.py

```diff
@@ -625,6 +684,8 @@ class SampleView(BaseView):
         "capture_kit",
     ]
     column_formatters = {
+        "application_version": view_application_version_link,
```

Comment:
Most of the important information is in the application table rather than the version. I would expect to be taken there when clicking on it